_data/projects/ideahub.yml

```diff
@@ -0,0 +1,12 @@
+name: IdeaHub
+desc: A collection of ideas and projects that contain from beginner to advance :octocat: 🎯🚀
```

Comment:
I think the `:` part of `:octocat:` here is not being handled correctly. If you wrap this in quote marks it should be handled as a string:

```suggestion
desc: "A collection of ideas and projects that contain from beginner to advance :octocat: 🎯🚀"
```

Reply:
Thanks @shiftkey